packages/block-library/src/image/editor.scss

```diff
@@ -107,6 +107,7 @@ figure.wp-block-image:not(.wp-block) {
 	.components-popover__content {
 		overflow: visible;
 		min-width: 260px;
+		min-height: 110px;
```

Comment:
The increased size doesn't look quite right when the input is focused and the tooltip isn't visible:
<img width="266" alt="Screenshot 2020-12-04 at 5 12 38 pm" src="https://user-images.githubusercontent.com/677833/101144900-f8d7a680-3653-11eb-8e8f-3f0522e2aa08.png">

My feeling is that increasing the size of the popover isn't needed, it looks ok to me when the tooltip is overlapping the edge of the popover:
<img width="268" alt="Screenshot 2020-12-04 at 5 09 54 pm" src="https://user-images.githubusercontent.com/677833/101144769-c9289e80-3653-11eb-967a-0f9564a0bb6d.png">

Reply:
Great, I concur @talldan. I'm removing the height adjustment in a moment.